charts/am-org-role-mapping-service/values.preview.template.yaml

```diff
@@ -77,7 +77,8 @@ orm:
 
 am-role-assignment-service:
   java:
-    image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
+    # temporary override to allow test of hmcts/am-role-assignment-service#2508: image: 'hmctspublic.azurecr.io/am/role-assignment-service:latest'
```

Comment:
This temporary override must be reversed before merger: but only after merger of hmcts/am-role-assignment-service#2508.

Reply:
Resolved in 3298881b03ddcd6de7222f417243dae74f7b9808.